HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealth.swift

```diff
@@ -104,6 +104,35 @@ public struct DataForReview {
         self.paymentComponentsController = PaymentComponentsController(giniHealth: self)
         self.paymentComponentsController.delegate = self
     }
+    
+    /**
+     Initializes a new instance of GiniHealth.
+     
+     This initializer creates a GiniHealth instance by first constructing a Client object with the provided client credentials (id, secret, domain)
+     
+     - Parameters:
+     - id: The client ID provided by Gini when you register your application. This is a unique identifier for your application.
+     - secret: The client secret provided by Gini alongside the client ID. This is used to authenticate your application to the Gini API.
+     - domain: The domain associated with your client credentials. This is used to scope the client credentials to a specific domain.
+     - pinningConfig: Configuration for certificate pinning. Format ["PinnedDomains" : ["PublicKeyHashes"]]
+     - logLevel: The log level. `LogLevel.none` by default.
+     */
+    public init(id: String,
+                secret: String,
+                domain: String,
+                apiVersion: Int = Constants.defaultVersionAPI,
```

Comment:
[clarification] Should `apiVersion` be included in the code documentation above?

Reply:
I don't think so. ApiVersion are just for internal use of the various of our SDKs. It's public because we use multiple SDKs, but the clients don't need to configure this.